tests/playwright-tests/tests/public/seoMetadataFiles.spec.ts

```diff
@@ -3,7 +3,10 @@ import environment from '@util/env';
 
 const { PUBLIC_URL, PROD_PUBLIC_URL } = environment;
 
-test.describe('SEO Metadata Files', () => {
+// Playwright finds two elements with getByText('...sitemal.xml') because the html span they're displayed
+// in contains the xml <loc> tag, which playwright then treats as another html element.
```

Comment:
Could use an XPath selector if there's an issue finding the correct element?

Also, it's weird that it's picking up a `span` because the sitemap doesn't actually contain this element!

Reply:
Yep was considering that, alternatively could actually make an API GET request to the `/sitemap.xml` route with content type set to xml, then parse the response and assert on it all properly that way. Just cautious not to spend too long on Playwright without approval to do so 

And, yeah, really no idea why it's getting the span!